src/rs_quark_assets/public/manifest.webmanifest

```
@@ -0,0 +1,23 @@
+{
+  "name": "QuarkJs",
+  "short_name": "QuarkJs",
```

```suggestion
  "name": "Quark",
  "short_name": "Quark",
```

quark.js is the name of our integrator script.
Quark is the name of our product. That should not be displayed on user's phones below their webapp icon.